### PR TITLE
ch-remote: Add support for vmm.ping

### PR DESCRIPTION
**Pull request by ZhangShuaiyi**
Add vmm-ping subcommand to ch-remote.

### CODE REVIEW

**Comment by liuw** [comment]
@ZhangShuaiyi there is no need to keep creating new PRs to fix issues. You can just do a force push the your source branch. Github will handle things correctly.

**Comment by ZhangShuaiyi** [comment]
>

File: src/bin/ch-remote.rs [hunk 1]
```diff
@@ -338,6 +339,11 @@ fn do_command(matches: &ArgMatches) -> Result<(), Error> {
         Some("counters") => {
             simple_api_command(&mut socket, "GET", "counters", None).map_err(Error::ApiClient)
         }
+        Some("vmm-ping") => {
+            let full_command = "vmm.ping";
+            simple_api_full_command_with_fds(&mut socket, "GET", &full_command, None, Vec::new())
+                .map_err(Error::ApiClient)
+        }
```
Comment (sboeuf) [inline src/bin/ch-remote.rs:346]: Please simplify as follows:
```suggestion
        Some("ping") => {
            simple_api_full_command(&mut socket, "GET", "vmm.ping", None)
                .map_err(Error::ApiClient)
        }
```

File: src/bin/ch-remote.rs [hunk 2]
```diff
@@ -675,7 +681,8 @@ fn main() {
             Command::new("create")
                 .about("Create VM from a JSON configuration")
                 .arg(Arg::new("path").index(1).default_value("-")),
-        );
+        )
+        .subcommand(Command::new("vmm-ping").about("Request the VMM API server status"));
```
Comment (sboeuf) [inline src/bin/ch-remote.rs:685]:
```suggestion
.subcommand(Command::new("ping").about("Ping the VMM to check for API server availability"));
```

**Review by sboeuf — CHANGES_REQUESTED** [review]

**Review by sboeuf — APPROVED** [review]

**Review by rbradford — APPROVED** [review]

**Review by likebreath — APPROVED** [review]

**Pull request merged**